lib/evss/disability_compensation_form/form526_to_lighthouse_transform.rb

```diff
@@ -606,8 +606,12 @@ def transform_direct_deposit(direct_deposit_source)
           direct_deposit.financial_institution_name = direct_deposit_source['bankName']
         end
         direct_deposit.account_type = direct_deposit_source['accountType'] if direct_deposit_source['accountType']
-        direct_deposit.account_number = direct_deposit_source['accountNumber'] if direct_deposit_source['accountNumber']
-        direct_deposit.routing_number = direct_deposit_source['routingNumber'] if direct_deposit_source['routingNumber']
+        if direct_deposit_source['accountNumber']
+          direct_deposit.account_number = direct_deposit_source['accountNumber'].strip
+        end
+        if direct_deposit_source['routingNumber']
+          direct_deposit.routing_number = direct_deposit_source['routingNumber'].strip
+        end
```

Comment:
Could both conditionals be simplified using the safe navigation operator, e.g., `direct_deposit.account_number = direct_deposit_source['accountNumber']&.strip`?

Reply:
Love it, thanks @freeheeling !